File: atinject-integration/src/main/resources/webapp/scripts/atinject/registration/IsUsernameAvailableResponse.js
define(["atinject/WebSocketResponse"],
function(WebSocketResponse){
    function IsUsernameAvailableResponse() {
        this._class = "org.atinject.api.registration.dto.IsUsernameAvailableResponse"
    };

    IsUsernameAvailableResponse.prototype = new WebSocketResponse();

        IsUsernameAvailableResponse.prototype.getAvailable = function() {
        return this.available;
    };

    IsUsernameAvailableResponse.prototype.setAvailable = function(available) {
        this.available = available;
        return this;
    };


    
    return IsUsernameAvailableResponse;
});

File: atinject-api-user-registration/src/main/java/org/atinject/api/registration/dto/IsUsernameAvailableResponse.java
package org.atinject.api.registration.dto;

import org.atinject.core.websocket.dto.WebSocketResponse;

public class IsUsernameAvailableResponse extends WebSocketResponse {

    private static final long serialVersionUID = 1L;

    private boolean available;
    
    public IsUsernameAvailableResponse(){
        super();
    }

    public boolean isAvailable() {
        return available;
    }

    public IsUsernameAvailableResponse setAvailable(boolean available) {
        this.available = available;
        return this;
    }
    
}
